docs/networking/diagnostics/deploy-ntopng-on-debian-8.md

```diff
@@ -0,0 +1,180 @@
+4. OpenVPN will be used as an example to demonstrate the capabilities of ntopng. You do not need to have it installed on your machine to complete this guide. However, if you are interested in learning more about OpenVPN, read the [Setting up a Hardened OpenVPN Server on Debian 8](/docs/networking/vpn/set-up-a-hardened-openvpn-server) guide.
+
+
+{: .note}
+> The steps in this guide require root privileges. Be sure to run the steps below as `root` or with `sudo`. For more information on privileges, see our [Users and Groups](/docs/tools-reference/linux-users-and-groups) guide.
+
+### Add the ntopng Repository
+
+    wget http://apt-stable.ntop.org/jessie/all/apt-ntop-stable.deb
+    dpkg -i apt-ntop-stable.deb
+    apt-get clean all
+    apt-get update
+
+### Update the System and Install ntopng
+
+Verify that your system is up to date. Install `ntopng` and supporting services:
+
+    update && apt upgrade -yuf
+    apt install pfring nprobe ntopng ntopng-data n2disk ethtool
+
+### Add System User for ntopng
+
+By default, ntopng is run as the user `nobody`. This is a good choice for daemons requiring minimal access to the system. However, ntopng installs files in directories to which the user `nobody` may not have access. Depending on how your system is configured, you can mitigate this by creating a new user for it:
+
+1.  Add user `ntopng`:
+
+		useradd -r -s /bin/false ntopng
+
+2.  Set permissions for user `ntopng` and installation files/directories as shown:
+
+		mkdir /var/tmp/ntopng
+		chown -R ntopng:ntopng /usr/share/ntopng /var/tmp/ntopng
+		chmod 1770 -R /var/tmp/ntopng
+		find /usr/share/ntopng -type d -print0 | xargs -0 chmod 744
+		find /usr/share/ntopng -type f -print0 | xargs -0 chmod 755
+
+## Configure ntopng
+
+ntopng has a built in web server and initializer. Configuration options can be defined in a config file or set from the command line during initialization on a per-use basis. If you use both the command line and the config file, ntopng will prioritize the settings in the config file.
+
+1.  Disable TCP segmentation offload. Replace `eth0` with your primary connection interface (usually `eth0`):
+
+        ethtool -K eth0 gro off gso off tso off
+
+2.  Check and verify that TCP segmentation is disabled:
+
+        ethtool -k eth0
+
+3.  Verify that the `tcp-segmentation-offload` section in the output matches the following:
+
+        tcp-segmentation-offload: off
+                tx-tcp-segmentation: off
+                tx-tcp-ecn-segmentation: off
+                tx-tcp-mangleid-segmentation: off
+                tx-tcp6-segmentation: off
+
+4.  Create `/etc/ntopng/ntopng.conf` and match the contents to the example listed below. Replace `example.com` with your Linode’s domain or public IP address. If needed, replace `eth0` with your primary network interface. If you want to review available configuration parameters, run `man ntopng` from the terminal.
```

Comment:
Hey Edward,

The "-m" option allows the user to specify local network addresses (i.e.  192.168.1.0/24). It wasn't included in my initial submission, so I don't know where it came from. Perhaps it found its way in by mistake. Here's what I think you're looking for. You'll want to place "example.com" in the "-w" option on line 102 like this: 

-w=example.com:3005

However, when I wrote this guide, I wanted to make the reader aware that if they did not own a domain, they could also substitute "example.com" with their Linode's public IP address. That was what [domain_or_ip] was supposed to signify. I hope that answers your question. If you have any other questions, just comment here and I'll be notified via email about it.